tests/test_parser.py

```diff
@@ -6,6 +6,7 @@
 
 from vyxal.lexer import *
 from vyxal.parse import *
+from vyxal.structure import *
```

Comment:
I think I accidentally added this line because of the previous `eval` stuff, I'll remove it later.